packages/fx-core/templates/plugins/resource/cicd_v2/azdo/README.md

```diff
@@ -0,0 +1,48 @@
+# How to use pre-cooked pipelines on Azure DevOps
```

Comment:
It's ok but I will suggest finding another location for these files. Maybe `fx-core/templates/components/cicd`?

Reply:
I found there are similia files in `fx-core/templates/plugins/resource/cicd/`. Is there any change?

Reply:
Good suggestion. Will update it in another PR since this PR is to be merged before pre-release.

Reply:
> I found there are similia files in `fx-core/templates/plugins/resource/cicd/`. Is there any change?

Yes - package.json doesn't exist in root, so the very first step is not `npm ci` any more.